_mobile/android.md

```diff
@@ -238,108 +273,9 @@ packagingOptions {
-
-Java’s primitive type byte is signed and java does not have unsigned 8 bit type. For dtype=uint8 java API uses java primitive `byte` that will be reinterpreted as uint8 on native side. On java side unsigned value of byte can be read as `byte & 0xFF`.
-
-#### Tensor Content Layout
-
-Tensor content is represented as a one dimensional array (buffer),
-where the first element has all zero indexes T\[0, ... 0\].
-
-Lets assume tensor shape is {d<sub>0</sub>, ... d<sub>n-1</sub>} and d<sub>n-1</sub> > 0.
-
-The second element will be T\[0, ... 1\] and the last one T\[d<sub>0</sub>-1, ... d<sub>n-1</sub> - 1\]
-
-Tensor has methods to check its dtype:
-```
-int dtype()
-```
-That returns one of the `DType` enum element:
-```
-DType.UINT8
-DType.INT8
-DType.INT32
-DType.FLOAT32
-DType.INT64
-DType.FLOAT64
-```
-
-The data of Tensor can be read as a java array:
-```
-byte[] getDataAsUnsignedByteArray()
-byte[] getDataAsByteArray()
-int[] getDataAsIntArray()
-long[] getDataAsLongArray()
-float[] getDataAsFloatArray()
-double[] getDataAsDoubleArray() 
-```
-These methods throw `IllegalStateException` if called for inappropriate dtype.
-
-### [`org.pytorch.IValue`](https://github.com/pytorch/pytorch/blob/master/android/pytorch_android/src/main/java/org/pytorch/IValue.java)
-
-IValue represents a TorchScript variable that can be one of the supported (by TorchScript) [types](https://pytorch.org/docs/stable/jit.html#types). 
-`IValue` is a tagged union. For every supported type it has a factory method, method to check the type and a getter method to retrieve a value.
-Getters throw `IllegalStateException` if called for inappropriate type.
-
-### [`org.pytorch.Module`](https://github.com/pytorch/pytorch/blob/master/android/pytorch_android/src/main/java/org/pytorch/Module.java)
-
-Module is a wrapper of torch.jit.ScriptModule (`torch::jit::script::Module` in PyTorch C++ API) which can be constructed with the factory method `load` providing absolute path to the file with serialized TorchScript.
-```
-IValue IValue.runMethod(String methodName, IValue... inputs)
-```
-for running a particular method of the script module.
-```
-IValue IValue.forward(IValue... inputs)
-```
-Shortcut to run 'forward' method.
-
-```
-IValue IValue.destroy()
-```
-Explicitly destructs native (C++) part of the Module, `torch::jit::script::Module`.
+## More Details
 
-As fbjni library destructs native part automatically when current `org.pytorch.Module` instance will be collected by Java GC, the instance will not leak if this method is not called, but timing of deletion and the thread will be at the whim of the Java GC. If you want to control the thread and timing of the destructor, you should call this method explicitly.
+More details about PyTorch Android API you can find in [docs](https://pytorch.org/docs/stable/packages.html) and [javadocs in source files of our API](https://github.com/pytorch/pytorch/tree/master/android/pytorch_android/src/main/java/org/pytorch).
```

Comment:
"You can find more details about the PyTorch Android API in the [Javadoc](link)."

I don't think we should link to the source files.  Any information that users need should be added to the docs.